### DISCUSSION TITLE
Downtime for https://myprivatesrx.us/

### DISCUSSION THREAD

**Discussion by hmcclurg** [searxng/searx-instances#353] [Report instance downtime]
My instance https://myprivatesrx.us/ unexpectedly went offline about 3 days ago which inconveniently was around when I left for vacation. I apologize for the downtime, and I will be working to resolve the issue later today.

**Comment by hmcclurg** [comment]
My instance is back up now

**Comment by unixfox** [comment]
Thank you for the notification.
